tinyagentos/routes/coding.py

```
@@ -132,4 +158,190 @@ async def delete_workspace(request: Request, workspace_id: str):
+        xy = line[:2]
+        path = line[3:].strip()
+        if " -> " in path:
+            path = path.split(" -> ", 1)[1]
+        if xy == "??":
+            untracked.add(path)
+        else:
+            tracked.append(path)
+
+    # Tracked: restore via git checkout
+    if tracked:
+        rc, _o, se = await _git(root, "checkout", "--", *tracked)
```

Ruff (0.15.17)

[warning] Unpacked variable `se` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)